src/tests/class_tests/smartpeak/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ configure_file(${PROJECT_SOURCE_DIR}/include/SmartPeak/test_config.h.in ${CONFIG
 #------------------------------------------------------------------------------
 # Boost
 #------------------------------------------------------------------------------
-find_package(boost COMPONENTS unit_test_framework)
+find_package(Boost COMPONENTS filesystem unit_test_framework)
```

Comment:
It will be nice when we can merge in C++17 to get rid of the `filesystem` once and for all.

Reply:
Would great and we can merely depend on the OpenMS prebuilt contribs for all the required libs for Windows CI.